### DISCUSSION TITLE
Pls advice how to clear trash

### DISCUSSION THREAD

**Discussion by pashkasan** [seaweedfs/seaweedfs#3701] [Q&A]
Hi, 
i have cluster of 4 servers with 2 disks each, storing json files to bucket , seems like on file replace its going to volume trash, files stored via s3 api 
on screenshot is one of servers, i marked what i want to fix
![image](https://user-images.githubusercontent.com/55329774/190511178-a0b55006-56e5-4440-b481-0d9264cb1bb2.png)

tried via weed shell on master
lock
volume.vacuum

but seems no effect

**Comment by chrislusf** [comment]
see `volume.vacuum -h`

**Reply by pashkasan** [reply]
tnx, working, 
but is there any option to avoid trash collection, ie delete trash on file replace
cant find this in docs
